### ISSUE TITLE
Disable tracking for logged in users

### ISSUE THREAD

**Issue by samnabi** [schnti/kirby-piwik#1]
It would be great to set a config value to disable tracking when a user is logged in.

**Comment by schnti** [comment]
Interesting idea.
I added another query and config variable.

```
if (!c::get('ka.piwik.trackingIfLoggedIn', true) && site()->user()) {
    return '';
}
```

**Comment by schnti** [comment]
#close

**Comment by schnti** [comment]
close #1

**Comment by schnti** [comment]
Closed #1.
